cgmanifest.json

```diff
@@ -16133,8 +16133,8 @@
         "type": "other",
         "other": {
           "name": "perl-Class-Data-Inheritable",
-          "version": "0.08",
-          "downloadUrl": "https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/Class-Data-Inheritable-0.08-clean.tar.gz"
+          "version": "0.09",
+          "downloadUrl": "https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/Class-Data-Inheritable-0.09-clean.tar.gz"
```

Comment:
It should actually contain upstream URL from which we download the source tarball, do some validations and then put it in to azurelinuxsrcstorage. Can you please correct it.